### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by simone-2014-maggio**
Added Blue Jay

### CODE REVIEW

**Comment by simone-2014-maggio** [comment]
ok

**Comment by spetterman66** [comment]
There's no way that'll work without the disk image being stored the server.

File: src/browser/main.js [hunk 1]
```diff
@@ -1123,8 +1123,16 @@
...
+        ];              {
+                id: "bluejay",
+                name: "Blue Jay",
+                fda: {
+                    url: host + "bj050.img",
+                    size: 11429888,
+                    async: false,
+                },
+                homepage: "https://archiveos.org/blue-jay/",
+             ];
```
Comment (copy) [inline src/browser/main.js:1135]: Could you properly format this?

**Comment by copy** [comment]
I will add bluejay manually.

**Pull request closed without merging**